targets/targets.json

```diff
@@ -2806,7 +2819,8 @@
             "CAN",
             "SERIAL_ASYNCH",
             "FLASH",
-            "MPU"
+            "MPU",
+            "WATCHDOG"
```

Comment:
This is not needed as target inherits from FAMILY_STM32 where WATCHDOG is already defined